legend-engine-xt-serviceStore-grammar/src/main/java/org/finos/legend/engine/language/pure/grammar/from/ServiceStoreParseTreeWalker.java

```diff
@@ -126,6 +159,32 @@ private ServiceStoreElement visitServiceStoreElement(ServiceStoreParserGrammar.S
         }
     }
 
+    private Pair<String, SecurityScheme> visitSecuritySchemeObject(ServiceStoreParserGrammar.SecuritySchemeObjectContext ctx)
+    {
+        SourceInformation sourceInformation = walkerSourceInformation.getSourceInformation(ctx);
+
+        ServiceStoreParserGrammar.SingleSecuritySchemeContext singleSecuritySchemeCtx = ctx.singleSecurityScheme();
+        SecuritySchemeSourceCode code = new SecuritySchemeSourceCode(
+                singleSecuritySchemeCtx.getText(),
+                singleSecuritySchemeCtx.securitySchemeType().getText(),
+                sourceInformation,
+                ParseTreeWalkerSourceInformation.offset(walkerSourceInformation, singleSecuritySchemeCtx.getStart())
+        );
+
+
+        List<IServiceStoreGrammarParserExtension> extensions = IServiceStoreGrammarParserExtension.getExtensions();
+        SecurityScheme securityScheme = IServiceStoreGrammarParserExtension.process(code, ListIterate.flatCollect(extensions, IServiceStoreGrammarParserExtension::getExtraSecuritySchemesParsers));
+
+        if (securityScheme == null)
```

Comment:
do we need this check, I see we already throwing exception in process itself

Reply:
removed